partner_secondhand_dealer/views/res_identification_type_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id='view_identification_type_form' model='ir.ui.view'>
+        <field name='name'>view.identification.type.form</field>
+        <field name='model'>identification.type</field>
+        <field name='type'>form</field>
+        <field name='arch' type='xml'>
+            <form string='Identification Type'>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id='view_identification_type_tree' model='ir.ui.view'>
+        <field name='name'>view.identification.type.tree</field>
+        <field name='model'>identification.type</field>
+        <field name='type'>tree</field>
+        <field name='arch' type='xml'>
+            <tree create="1">
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+
+    <record id='action_identification_type' model='ir.actions.act_window'>
+        <field name='name'>Identification Types</field>
+        <field name='res_model'>identification.type</field>
+        <field name='view_mode'>tree,form</field>
+        <field name="view_id" ref='view_identification_type_tree' />
+    </record>
+
+    <menuitem
+        id="menu_vendor_in_config_purchase"
+        name="Vendors"
+        parent="purchase.menu_purchase_config"
+        groups="base.group_no_one"
+        sequence="40"
+    />
```

Comment:
Can we move this one to `/data/menuitem_data.xml`?

We prefer to keep "node" type menu items and the ones that refer to existing actions in this file, just for better organization.

ref. https://github.com/qrtl/rmm-custom/blob/b4afe29d2abb91e715b455be6d95f9729da36b2b/product_auction/data/menuitem_data.xml